packages/kolibri/utils/i18n.js

```diff
@@ -244,38 +250,19 @@ export function i18nSetup(skipPolyfill = false) {
 
   // If the browser doesn't support the Intl polyfill, we retrieve that and
   // the modules need to wait until that happens.
-  return new Promise((resolve, reject) => {
-    if (Object.prototype.hasOwnProperty.call(global, 'Intl') || skipPolyfill) {
-      _setUpVueIntl();
-      resolve();
-    } else {
-      Promise.all([
-        new Promise(res => {
-          require.ensure(
-            ['intl'],
-            require => {
-              res(() => require('intl'));
-            },
-            'intl',
-          );
-        }),
-        importIntlLocale(currentLanguage),
-      ]).then(
-        // eslint-disable-line
-        ([requireIntl, requireIntlLocaleData]) => {
-          requireIntl(); // requireIntl must run before requireIntlLocaleData
-          requireIntlLocaleData();
-          _setUpVueIntl();
-          resolve();
-        },
-        error => {
-          logging.error(error);
-          logging.error('An error occurred trying to setup Internationalization', error);
-          reject();
-        },
-      );
+  if (Object.prototype.hasOwnProperty.call(global, 'Intl') || skipPolyfill) {
```

Comment:
Now seemed as good a time as any to change this function to async/await to avoid it being so verbose.